### DISCUSSION TITLE
Somethings Wrong With My Kade Engine

### DISCUSSION THREAD

**Discussion by Modingunreal** [KadeArchive/Kade-Engine#2847] [Q&A]
Pls Help Me With This

Problem Lime Test Windows
![Capture3](https://user-images.githubusercontent.com/98867791/153703922-c44a95f6-4462-48d1-a1a2-59ea951e7055.PNG)

**Comment by ghost** [comment]
Did you install the requirements?

**Comment by ghost** [comment]
https://github.com/KadeDev/Kade-Engine/blob/stable/docs/troubleshooting.md#type-not-found--statepointer

**Comment by AwesomeWithRex** [comment]
reinstall everything on the building.md
